docs/standardisation-scope.adoc

```diff
@@ -200,20 +224,17 @@ It is the goal of this project to create a good and feature-rich standard langua
 === Functions
 
 * `all()`
-* `allShortestPaths()`
 * `any()`
 * `distance()`
+* `exists()`
```

Comment:
Please see my earlier comment about the two forms of `exists()`

Reply:
It's not included as implemented in Neo4j, but it is included as specified in the EXISTS CIP.